src/helpers.js

```diff
@@ -23,5 +27,5 @@ export const stringifyQuery = (params = {}, prefix) => {
     return `${key}=${encodeURIComponent(value)}`;
   });
 
-  return [].concat.apply([], query).join('&');
+  return query.join('&');
```

Comment:
Out of curiosity I was investigating, and from what I can tell, in the `RestApiClient.js` file, where `url.search = stringifyQuery(query);` is called, I'm guessing assigning the string to the `url.search` property automagically prepends the `?` to the string?